src/components/LoadingComment/index.js

```diff
@@ -26,7 +26,7 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blackgrey` to `COLORS.gray400` for the `color` property in the `username` style is also a simple color change and should not affect the functionality of the component.
```diff
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->